### PR TITLE
Create model card

### PR DESCRIPTION
**Pull request by mrm8488**
Create Model card for distilroberta-base-finetuned-sentiment

### CODE REVIEW

File: model_cards/mrm8488/distilroberta-base-finetuned-sentiment/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# DistilRoBERTa + Sentiment Analysis 😂😢😡😃😯
+
+This in an adaption version of [@omarsar0](https://twitter.com/omarsar0) [tutorial](https://t.co/WMnATW0Hwf?amp=1)
+He explain everything so detailed and provided the dataset. I just changed some parameters and created the ```config.json```file to upload it to [🤗Transformers HUB](https://huggingface.co/) 
+
+
+In this tutorial, He shows to fine-tune a language model (LM) for **emotion classification** with code adapted from this [tutorial](https://zablo.net/blog/post/custom-classifier-on-bert-model-guide-polemo2-sentiment-analysis/) by MARCIN ZABŁOCKI.
```
Comment (julien-c) [inline model_cards/mrm8488/distilroberta-base-finetuned-sentiment/README.md:7]:
```suggestion
In this tutorial, he shows how to fine-tune a language model (LM) for **emotion classification** with code adapted from this [tutorial](https://zablo.net/blog/post/custom-classifier-on-bert-model-guide-polemo2-sentiment-analysis/) by MARCIN ZABŁOCKI. 
```

File: model_cards/mrm8488/distilroberta-base-finetuned-sentiment/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# DistilRoBERTa + Sentiment Analysis 😂😢😡😃😯
+
+This in an adaption version of [@omarsar0](https://twitter.com/omarsar0) [tutorial](https://t.co/WMnATW0Hwf?amp=1)
+He explain everything so detailed and provided the dataset. I just changed some parameters and created the ```config.json```file to upload it to [🤗Transformers HUB](https://huggingface.co/)
```
Comment (julien-c) [inline model_cards/mrm8488/distilroberta-base-finetuned-sentiment/README.md:4]:
```suggestion
He explains everything so detailed and provided the dataset. I just changed some parameters and created the ```config.json```file to upload it to [🤗Transformers HUB](https://huggingface.co/) 
```

**Review by julien-c — APPROVED** [review]

**Comment by julien-c** [comment]
Great!

**Review by mrm8488 — COMMENTED** [review]
Thanks, @julien-c!

**Comment by mrm8488** [comment]
You do not rest even on Saturday! 🦾

**Pull request merged**